pages/partials/question.ejs

```diff
@@ -41,19 +41,33 @@
       <div class="collapse" id="issue-course-data-<%= iIssue %>">
         <pre class="bg-dark text-white rounded p-3"><%= JSON.stringify(issue.course_data, null, '    ') %></pre>
       </div>
-      <% if (is_administrator) { %>
+      <% if ((devMode || authz_data.has_instructor_view || is_administrator) && (is_administrator)) { %>
       <p><strong>System data:</strong>
         <button type="button" class="btn btn-xs btn-secondary" data-toggle="collapse" href="#issue-system-data-<%= iIssue %>" aria-expanded="false" aria-controls="#issue-system-data-<%= iIssue %>">
           Show/hide
         </button>
       </p>
+      <% } %>
+      <p><strong>Issue status:</strong>
+        <% if (issue.open) { %>
+          <span type="button" class="badge badge-danger" data-toggle="collapse" href="#">
+            Open
+          </span>
+          <% } else { %>
+            <span type="button" class="badge badge-success" data-toggle="collapse" href="#">
+              Closed
+            </span>
+          <% } %>
+      </p>
```

Comment:
Can we put the issue status in the table above? That table is used for standard issue data, and I think "status" is like this. It would also avoid the need to change around the authorization checking.